launcher/minecraft/MinecraftInstance.cpp

```diff
@@ -925,7 +925,10 @@ QString MinecraftInstance::getStatusbarDescription()
     if(m_settings->get("ShowGameTime").toBool())
     {
         if (lastTimePlayed() > 0) {
-            description.append(tr(", last played for %1").arg(Time::prettifyDuration(lastTimePlayed())));
+            QDateTime lastLaunchTime = QDateTime::fromMSecsSinceEpoch(lastLaunch());
+            description.append(tr(", last played at %1 for %2")
+                                   .arg(QLocale::system().toString(lastLaunchTime, QLocale::ShortFormat))
```

Comment:
We should probably use the locale set in the settings instead.

```suggestion
                                   .arg(QLocale().toString(lastLaunchTime, QLocale::ShortFormat))
```

Reply:
You're the boss, i've just thought in case the user prefers to use English as a Language but use the Locale of another.